agent/engine/docker_task_engine.go

```diff
@@ -229,8 +230,8 @@ func (engine *DockerTaskEngine) MustInit(ctx context.Context) {
 	defer engine.mustInitLock.Unlock()
 
 	errorOnce := sync.Once{}
-	taskEngineConnectBackoff := utils.NewSimpleBackoff(200*time.Millisecond, 2*time.Second, 0.20, 1.5)
-	utils.RetryWithBackoff(taskEngineConnectBackoff, func() error {
+	taskEngineConnectBackoff := retry.NewExponentialBackoff(200*time.Millisecond, 2*time.Second, 0.20, 1.5)
```

Comment:
Can we move the values here to a constant as well, will give a better understanding of what they are.

Reply:
Sure, make sense :)